recipes/r-rinside/build.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+if [[ $target_platform =~ linux.* ]] || [[ $target_platform == win-32 ]] || [[ $target_platform == win-64 ]] || [[ $target_platform == osx-64 ]]; then
+  export DISABLE_AUTOBREW=1
+  $R CMD INSTALL --build .
+else
+  mkdir -p $PREFIX/lib/R/library/RInside
+  mv * $PREFIX/lib/R/library/RInside
+  if [[ $target_platform == osx-64 ]]; then
+    pushd $PREFIX
+      for libdir in lib/R/lib lib/R/modules lib/R/library lib/R/bin/exec sysroot/usr/lib; do
```

Comment:
Quick note about the helper script:

this section appears to concern actually copying and relocating already built binaries from the tarball.

is this something that we should be doing?

Reply:
Yes, its for binary repacking, but afaik currently not used here...

Reply:
Thanks. Do you know if it is used for any R package on conda-forge?